src/Translator/src/TranslationsDumper.php

```diff
@@ -182,18 +185,4 @@ private function getLocaleFallbacks(MessageCatalogueInterface ...$catalogues): a
 
         return $localesFallbacks;
     }
-
-    private function generateConstantName(string $translationId): string
-    {
-        $translationId = s($translationId)->ascii()->snake()->upper()->replaceMatches('/^(\d)/', '_$1')->toString();
-        $prefix = 0;
-        do {
-            $constantName = $translationId.($prefix > 0 ? '_'.$prefix : '');
-            ++$prefix;
-        } while ($this->alreadyGeneratedConstants[$constantName] ?? false);
-
-        $this->alreadyGeneratedConstants[$constantName] = true;
-
-        return $constantName;
-    }
```

Comment:
I didn't make Blackfire profiles, but removing this code will surely help to improve performances when dealing with a ton of translations.